modules/va_facilities/app/controllers/va_facilities/v1/nearby_controller.rb

```diff
@@ -17,31 +16,34 @@ class NearbyController < ApplicationController
       skip_before_action(:authenticate)
       before_action :set_default_format
       before_action :set_facility_type
+      before_action :set_default_drivetime
       before_action :validate_params, only: [:index]
 
+      PER_PAGE = 20
+
       REQUIRED_PARAMS = {
         address: %i[street_address city state zip].freeze,
         lat_lng: %i[lat lng].freeze
       }.freeze
-      QUERY_INFO = {
-        address: NearbyFacility.method(:query),
-        lat_lng: NearbyFacility.method(:query_by_lat_lng)
-      }.freeze
 
       def index
-        query_method = get_query_method(params)
-        params_hash = params.permit!.to_h.symbolize_keys
-        resource = query_method.call(params_hash).paginate(page: params[:page],
-                                                           per_page: params[:per_page] || NearbyFacility.per_page)
+        lat_lng = get_lat_lng(params)
+        eligible_facilities = Facilities::VHAFacility.with_services(params[:services]) if params[:services]
+
+        resource = if lat_lng.present?
```

Comment:
I think I would prefer a more specific variable name here, perhaps `drivetime_bands` or something instead of `resource`

Reply:
I'm going with `bands`